### PR TITLE
chore(AHWR-231): improved logging, improved service startup and dependency update

### PR DESCRIPTION
**Pull request by bendef-24**

### CODE REVIEW

File: test/integration/narrow/app/logger.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+describe('Logger test', () => {
```
Comment (mattyod) [inline test/integration/narrow/app/logger.test.js:1]: Not sure I see much value in this test.

File: app/index.js [hunk 2]
```diff
@@ -2,13 +2,15 @@ import { setup } from './insights.js'
 import 'log-timestamp'
 import { startSfdMessageReceiver, stopSfdMessageReceiver } from './messaging/index.js'
 import server from './server.js'
+import logger from './logger.js'
 
 const init = async () => {
-  await startSfdMessageReceiver(server.logger)
+  await server.register([logger])
```
Comment (mattyod) [inline app/index.js:8]: We tend to initialise server plugins in the server file along with the rest of it's bootstrapping. That way you can initialise a server for testing and it'll be sure it's the same as what we use when running the app. Means you wont have to keep stubbing the logger methods.

It's configured to log silently during tests 🙊 https://github.com/DEFRA/ffc-ahwr-sfd-messaging-proxy/blob/main/app/logger.js#L4

File: test/integration/narrow/app/logger.test.js (hunk 1 above)
Comment (bendef-24) [reply test/integration/narrow/app/logger.test.js:1]: I do agree, but logger is mocked everywhere else and without this test the coverage dropped below 90%. I could write a test that uses the real logger if you want?

File: app/index.js (hunk 2 above)
Comment (bendef-24) [reply app/index.js:8]: Might be worth a quick call this one, was hoping you'd highlight this.

**Pull request merged**